openfl/component/aggregator/aggregator.py

```diff
@@ -255,18 +290,31 @@ def _save_model(self, round_number, file_path):
             for k, v in og_tensor_dict.items()
         ]
         tensor_dict = {}
+        tensor_tuple_dict = {}
         for tk in tensor_keys:
             tk_name, _, _, _, _ = tk
-            tensor_dict[tk_name] = self.tensor_db.get_tensor_from_cache(tk)
+            print(f"transaction - tk is {tk} len {len(tk)} tk_name is {tk_name}")
```

Comment:
nit - use logger here too

Reply:
This will pollute the logs. I would suggest changing this to `logger.debug`

Reply:
sure, all the non log prints were for local testing (should have been removed prior publishing the pr)